libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SmartSyncPlugin.java

```diff
@@ -288,7 +295,7 @@ private SyncManager getSyncManager(boolean isGlobal,String storeName) {
      * @param arg0 first argument passed in plugin call
      * @return
      */
-    private static SmartStore getSmartStore(JSONObject arg0) {
+    private static SmartStore getSmartStore(JSONObject arg0) throws Exception {
```

Comment:
Could we only have the next two methods in SmartStorePlugin.java and have SmartSyncPlugin.java call them there - like we do in the React bridges?